dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/trackedentity/hibernate/HibernateTrackedEntityInstanceStore.java

```diff
@@ -929,7 +929,8 @@ private void joinAttributeValueWithoutQueryParameter( StringBuilder attributes,
                     .append( SPACE )
                     .append( filter.getSqlOperator() )
                     .append( SPACE )
-                    .append( StringUtils.lowerCase( filter.getSqlFilter( filter.getFilter() ) ) );
+                    .append( StringUtils
+                        .lowerCase( filter.getSqlFilter( statementBuilder.encode( filter.getFilter(), false ) ) ) );
```

Comment:
This is 4 nested method calls on one line, maybe worth a variable for readability?

Reply:
Done!